lte/gateway/c/session_manager/LocalEnforcer.h

```diff
@@ -39,6 +39,8 @@
 namespace magma {
 using std::experimental::optional;
 
+using shards_ = std::vector<int>;
```

Comment:
this should be proper initialized

Maybe you will also need functions to grow and shrink the vector.